### PR TITLE
intent classification using rasa-nlu

### PR DESCRIPTION
**Pull request by gautamig54**
I have used rasa-nlu toolkit for intent classification. I have trained the rasa model for 14 intents(greet, goodbye, affirm, deny, mood_great, mood_unhappy, currency, transfer_money, show_credit_cards, type_weather, temperature_weather, ask_forecast_weather). We can train the model just by adding intents to the nlu.md file. 
I have added the trained model here. I have linked the model to the chatbot.py and the model will predict the intent of content spoken/written by the user.

### CODE REVIEW

**Comment by gautamig54** [comment]
Should I upload the nlu.md file as well, which is the training data for rasa containing the intents?

File: chatbot.py [hunk 1]
```diff
@@ -93,6 +104,8 @@
                 print("Could not understand audio")
                 engine.say('I didnt get that. Rerun the code')
                 engine.runAndWait()
+    intent = interpreter.parse(translate)
+    print(intent['intent'])
```
Comment (satyammittal) [inline chatbot.py:108]: no need to print intent.

**Review by satyammittal — COMMENTED** [review]
I don't see any change on replacing rule based with intent you get from model.

File: chatbot.py (hunk 1 above)
Comment (gautamig54) [reply chatbot.py:108]: I just added it to check the intent. I will remove it.

**Pull request closed without merging**